arangod/RestHandler/RestEdgesHandler.cpp

```diff
@@ -261,11 +278,23 @@ bool RestEdgesHandler::readEdgesForMultipleVertices() {
 
   std::unordered_set<std::string> foundEdges;
 
+  // Check if dirty reads are allowed:
+  bool found = false;
+  std::string const& val =
+      _request->header(StaticStrings::AllowDirtyReads, found);
+  bool allowDirtyReads = false;
+  if (found && StringUtils::boolean(val)) {
+    // This will be used in `createTransaction` below, if that creates
+    // a new transaction. Otherwise, we use the default given by the
+    // existing transaction.
+    allowDirtyReads = true;
+  }
```

Comment:
```suggestion
  // This will be used in `createTransaction` below, if that creates
  // a new transaction. Otherwise, we use the default given by the
  // existing transaction.
  allowDirtyReads = _request->parsedValue(StaticStrings::AllowDirtyReads, /*default*/ false);
```